types/three/examples/jsm/loaders/GLTFLoader.d.ts

```diff
@@ -131,6 +131,8 @@ export class GLTFParser {
 }
 
 export interface GLTFLoaderPlugin {
+    constructor(parser: GLTFParser);
+    get name(): string;
```

Comment:
nit: my personal preference is to use a normal property when it could be either, unless I'm missing a reason to do it as a getter.
```suggestion
    readonly name: string;
```